schema/pet-insurance.json

```diff
@@ -152,17 +201,649 @@
                 "petType": {
                     "type": "string",
                     "description": "",
-                    "example": "dog"
+                    "example": "dog",
+                    "enum": [
+                        "dog",
+                        "cat"
+                    ]
                 },
                 "petGender": {

```

Comment:
The name of the key is "petGender", which is inside "Pets". This property is clearly part of a pet, so it doesn't need to be prefixed with "pet". Can it be just "gender"?

Reply:
I'll fix.